Tasks/Ruban/task1.py

```diff
@@ -0,0 +1,11 @@
+amount = float(input('Please enter amount of deposit, USD \n'))
+term = float(input('Please enter term of deposit, years \n'))
+interest = float(input('Please enter interest rate, % \n'))
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point